enterprise/ha/src/test/java/org/neo4j/test/ha/ClusterManager.java

```diff
@@ -966,7 +966,7 @@ private void startMember( InstanceId serverId ) throws URISyntaxException, IOExc
                 builder.setConfig( ClusterSettings.cluster_name, name );
                 builder.setConfig( ClusterSettings.initial_hosts, initialHosts.toString() );
                 builder.setConfig( ClusterSettings.server_id, serverId + "" );
-                builder.setConfig( ClusterSettings.cluster_server, "0.0.0.0:" + clusterPort );
+                builder.setConfig( ClusterSettings.cluster_server, clusterUri.getHost() + ":" + clusterPort );
```

Comment:
I am wondering if this is necessary. The tests should continue to work with ips regardless of this change, *requiring* hostnames is a behavioral change which is not expected.